[2023-09-27T17:21:29.896-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.download manual__2023-09-27T21:16:18.707894+00:00 [queued]>
[2023-09-27T17:21:29.906-0400] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Server_Access_Log_Processing.download manual__2023-09-27T21:16:18.707894+00:00 [queued]>
[2023-09-27T17:21:29.907-0400] {taskinstance.py:1359} INFO - Starting attempt 2 of 2
[2023-09-27T17:21:30.017-0400] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): download> on 2023-09-27 21:16:18.707894+00:00
[2023-09-27T17:21:30.021-0400] {standard_task_runner.py:57} INFO - Started process 68831 to run task
[2023-09-27T17:21:30.023-0400] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'ETL_Server_Access_Log_Processing', 'download', 'manual__2023-09-27T21:16:18.707894+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Server_Access_Log_Processing.py', '--cfg-path', '/tmp/tmpm4ncvdy9']
[2023-09-27T17:21:30.024-0400] {standard_task_runner.py:85} INFO - Job 7: Subtask download
[2023-09-27T17:21:30.171-0400] {task_command.py:415} INFO - Running <TaskInstance: ETL_Server_Access_Log_Processing.download manual__2023-09-27T21:16:18.707894+00:00 [running]> on host erosennnin-TUF-Gaming-FX505DU-FX505DU
[2023-09-27T17:21:30.309-0400] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ramesh@somemail.com' AIRFLOW_CTX_DAG_OWNER='Ramesh Sannareddy' AIRFLOW_CTX_DAG_ID='ETL_Server_Access_Log_Processing' AIRFLOW_CTX_TASK_ID='download' AIRFLOW_CTX_EXECUTION_DATE='2023-09-27T21:16:18.707894+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-27T21:16:18.707894+00:00'
[2023-09-27T17:21:30.310-0400] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-27T17:21:30.310-0400] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'wget -O /home/erosennnin/Programing/Apache Airflow/downloads/web-server-access-log.txt "https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBM-DB0250EN-SkillsNetwork/labs/Apache%20Airflow/Build%20a%20DAG%20using%20Airflow/web-server-access-log.txt"']
[2023-09-27T17:21:30.319-0400] {subprocess.py:86} INFO - Output:
[2023-09-27T17:21:30.321-0400] {subprocess.py:93} INFO - --2023-09-27 17:21:30--  http://airflow/downloads/web-server-access-log.txt
[2023-09-27T17:21:30.322-0400] {subprocess.py:93} INFO - Resolving airflow (airflow)... failed: Name or service not known.
[2023-09-27T17:21:30.322-0400] {subprocess.py:93} INFO - wget: unable to resolve host address ‘airflow’
[2023-09-27T17:21:30.322-0400] {subprocess.py:93} INFO - --2023-09-27 17:21:30--  https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBM-DB0250EN-SkillsNetwork/labs/Apache%20Airflow/Build%20a%20DAG%20using%20Airflow/web-server-access-log.txt
[2023-09-27T17:21:30.616-0400] {subprocess.py:93} INFO - Resolving cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud (cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud)... 169.45.118.108
[2023-09-27T17:21:30.820-0400] {subprocess.py:93} INFO - Connecting to cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud (cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud)|169.45.118.108|:443... connected.
[2023-09-27T17:21:31.844-0400] {subprocess.py:93} INFO - HTTP request sent, awaiting response... 200 OK
[2023-09-27T17:21:31.844-0400] {subprocess.py:93} INFO - Length: 8351 (8,2K) [text/plain]
[2023-09-27T17:21:31.845-0400] {subprocess.py:93} INFO - Saving to: ‘/home/erosennnin/Programing/Apache’
[2023-09-27T17:21:31.845-0400] {subprocess.py:93} INFO - 
[2023-09-27T17:21:31.845-0400] {subprocess.py:93} INFO -      0K ........                                              100%  168M=0s
[2023-09-27T17:21:31.845-0400] {subprocess.py:93} INFO - 
[2023-09-27T17:21:31.845-0400] {subprocess.py:93} INFO - 2023-09-27 17:21:31 (168 MB/s) - ‘/home/erosennnin/Programing/Apache’ saved [8351/8351]
[2023-09-27T17:21:31.846-0400] {subprocess.py:93} INFO - 
[2023-09-27T17:21:31.846-0400] {subprocess.py:93} INFO - FINISHED --2023-09-27 17:21:31--
[2023-09-27T17:21:31.846-0400] {subprocess.py:93} INFO - Total wall clock time: 1,5s
[2023-09-27T17:21:31.846-0400] {subprocess.py:93} INFO - Downloaded: 1 files, 8,2K in 0s (168 MB/s)
[2023-09-27T17:21:31.847-0400] {subprocess.py:97} INFO - Command exited with return code 4
[2023-09-27T17:21:31.860-0400] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/erosennnin/anaconda3/envs/coursera/lib/python3.11/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 4.
[2023-09-27T17:21:31.866-0400] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=ETL_Server_Access_Log_Processing, task_id=download, execution_date=20230927T211618, start_date=20230927T212129, end_date=20230927T212131
[2023-09-27T17:21:31.981-0400] {standard_task_runner.py:104} ERROR - Failed to execute job 7 for task download (Bash command failed. The command returned a non-zero exit code 4.; 68831)
[2023-09-27T17:21:32.000-0400] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-27T17:21:32.013-0400] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
